### ISSUE TITLE
Stepped number controllers generating extra onChange events

### ISSUE THREAD

**Issue by Sceat** [georgealways/lil-gui#125]
to reproduce:

```js
      gui
        .add({ count: 1 }, 'count', 1, 5, 1)
        .name('count')
        .onChange(count => {
          console.log('change', count)
        })
```

_Originally posted by @Sceat in https://github.com/georgealways/lil-gui/issues/124#issuecomment-1807726827_

**Comment by georgealways** [comment]
Thanks for this. I'm able to reproduce, but to confirm, it seems like this only affects number sliders with a step?

**Comment by Sceat** [comment]
most probably yes

**Comment by psnet** [comment]
I can confirm this right in the off docs example:

https://lil-gui.georgealways.com/#Guide#Numbers-and-Sliders

- If you put cursor in field
![зображення](https://github.com/georgealways/lil-gui/assets/1023918/1c436447-dada-49eb-b94c-c439e55df3b8)

- scroll mouse wheel in any direction, example up
![зображення](https://github.com/georgealways/lil-gui/assets/1023918/74a2d632-a5d3-4b91-b320-b67ffdadc07e)

- step will be not `10` as set by `step()` method, but `1130`
- if scroll mouse wheel backward it will be default value again (i.e. offset / step is not calculated correctly, but calc for + and - is ok)
![зображення](https://github.com/georgealways/lil-gui/assets/1023918/5f515ebc-c782-4f99-8b33-5ea5640693b7)

**Comment by psnet** [comment]
I can confirm this for number input fields and for sliders too

**Comment by georgealways** [comment]
Hi @psnet, I'm not sure if this was intentional but it appears to be a duplicate of your comment on another issue: https://github.com/georgealways/lil-gui/issues/46#issuecomment-1972019070

As for extra onChange events, there will be a fix for this issue in the next release.

**Comment by georgealways** [comment]
Hi all, this issue is fixed as of 0.19.2
